spython/version.py

```diff
@@ -5,7 +5,7 @@
 # with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
-__version__ = "0.3.1"
+__version__ = "0.3.2"
```

Comment:
This should be `0.3.11`

Not perfect semver, but allows for tiny changes like this one :)

Reply:
Updated :)